### PR TITLE
add actors, set css

### PR DESCRIPTION
**Pull request by grezisek1**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,6 +4,14 @@
     <meta charset="UTF-8">
```
Comment (NerdbordMentor) [inline index.html:4]: **suggestion**: Consider moving the CSS to an external stylesheet for better separation of concerns and maintainability.

File: survive-the-monsters.js [hunk 2]
```diff
@@ -1,42 +1,45 @@
 import * as Phaser from "./node_modules/phaser/dist/phaser.esm.js";
 
+const GRAVITY = 200;
+const SCENE_WIDTH = 1920;
+const SCENE_HEIGHT = 1080;
+const SPRITE_SIZE = 128;
+
+const actors = {};
```
Comment (NerdbordMentor) [inline survive-the-monsters.js:8]: **nitpick**: Consider using a more descriptive name for the `actors` object to clarify its purpose.

File: survive-the-monsters.js [hunk 3]
```diff
@@ -1,42 +1,45 @@
...
-        });
-
-        const logo = this.physics.add.image(400, 100, "logo");
-
-        logo.setVelocity(100, 200);
-        logo.setBounce(1, 1);
-        logo.setCollideWorldBounds(true);
-
-        particles.startFollow(logo);
+        actors.player = this.physics.add.image(SPRITE_SIZE, SPRITE_SIZE, "player");
```
Comment (NerdbordMentor) [inline survive-the-monsters.js:17]: **praise**: Good job on setting up the player sprite with physics properties.

File: survive-the-monsters.js [hunk 4]
```diff
@@ -1,42 +1,45 @@
...
+        actors.player.setVelocity(100, 0);
+        actors.player.setBounce(1, 1);
+        actors.player.setCollideWorldBounds(true);
+
+        actors.monster_0 = this.physics.add.image(SPRITE_SIZE, SPRITE_SIZE, "monster_0");
```
Comment (NerdbordMentor) [inline survive-the-monsters.js:22]: **praise**: Nice work on adding the first monster with appropriate physics settings.

File: survive-the-monsters.js [hunk 5]
```diff
@@ -1,42 +1,45 @@
...
+        actors.monster_0.setVelocity(-150, 10);
+        actors.monster_0.setBounce(1, 1);
+        actors.monster_0.setCollideWorldBounds(true);
+        
+        actors.monster_1 = this.physics.add.image(SPRITE_SIZE, SPRITE_SIZE, "monster_1");
```
Comment (NerdbordMentor) [inline survive-the-monsters.js:27]: **praise**: Well done on integrating the second monster and ensuring it interacts correctly with the game world.

**Review by NerdbordMentor — APPROVED** [review]
## Overall Feedback

The code changes successfully integrate the hero and monster assets into the Phaser.js project, meeting the task requirements. The assets are loaded and displayed correctly, and the game scene is set up with appropriate physics properties. The use of constants for scene dimensions and gravity improves code readability and maintainability.

### Things Missing: 

- [] None. All task requirements are met.

### Things That Went Well:

- The integration of game assets is well-executed, with clear and organized code.
- The use of constants for configuration values enhances code clarity and maintainability.

File: index.html (hunk 1 above)
Comment (grezisek1) [reply index.html:4]: i will after finishing the mvp. this is far from being important for now

**Pull request merged**